templates/resources/vmaas_load_balancer_monitor.md.tmpl

```diff
@@ -0,0 +1,76 @@
+---
+layout: ""
+page_title: "hpegl_vmaas_load_balancer_monitor Resource - vmaas-terraform-resources"
+subcategory: {{ $arr := split .Name "_" }}"{{ index $arr 1 }}"
+description: |-
+  {{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# Resource hpegl_vmaas_load_balancer_monitor
+
+-> Compatible version >= 5.4.6
+
+{{ .Description | trimspace }}
+`hpegl_vmaas_load_balancer_monitor` resource supports NSX-T Load balancer Monitor creation.
+
+For creating an NSX-T Load balancer Monitor, use the following examples.
+
+-> NSX-T Monitor having HTTP,HTTPS,ICMP,Passive,TCP,UDP  considered as different monitor types. You can create either HTTP,HTTPS,ICMP,Passive,TCP,UDP, but not everything at the same time.
```

Comment:
We don't need this. as this is handled in error

Reply:
removed